### ISSUE TITLE
Computers not waking if not shutdown be WOL

### ISSUE THREAD

**Issue by dcnewton76** [basildane/WakeOnLAN#114]
Hi,
If a computer is shutdown in the normal way it seems that WOL cannot wake it up.

**Comment by dcnewton76** [comment]
Yep, 7 are working but the rest of the room are not. it is when the computer is shutdown via the start menu that WOL will not wake it up.

**Comment by basildane** [comment]
Is the NIC light blinking after the computer is shutdown?

**Comment by dcnewton76** [comment]
hi, yes the NIC light is still on

**Comment by dcnewton76** [comment]
We do not have WOL installed on the remote computers, is this necessary for it to work properly?

**Comment by basildane** [comment]
You do not need to install anything on the remote computers.

I only suggested that for testing reasons.  There is a "listener" function in WOL that will show you if the computer is receiving WOL packets.

**Comment by Moopere** [comment]
> Yep, 7 are working but the rest of the room are not. it is when the computer is shutdown via the start menu that WOL will not wake it up.

Mmm.  Windows 10?

The start menu shutdown function changed in one of the more recent Win10 builds and it is no longer a true shutdown.  You will note that many Windows apps which were open before shutting down are open again when you start up.

Try shutting down this problematic machine using the command line:

shutdown /s /t 0

This should be equivalent to the long standing 'old' way to shut the computer down and roughly equivalent to a power-off event followed by a power-on event ... where WOL should be armed, waiting for a magic packet to arrive.  Mostly, though not always on very modern machines, this is indicated by your NIC activity LED blinking even though your PC is in S5 shutdown.
